element_miniscope/readers/miniscope_analysis_loader.py

```diff
@@ -70,10 +71,10 @@ def segmentation_channel(self):
 
      def extract_masks(self):
           masks = []
-          for i in range(int(self.mat_ms['ms']['numNeurons'][0,0])):
-               center_y, center_x = scipy.ndimage.measurements.center_of_mass(self.mat_sfp[i,:,:])
-               xpix, ypix, weights = scipy.sparse.find(self.mat_sfp[i,:,:])
-                
+          for i in tqdm(range(int(self.mat_ms['ms']['numNeurons'][0,0]))):
```

Comment:
not a big deal but the display may get mixed up with nested `tqdm` (when we do `.populate(display_progress=True)`)

Reply:
Thanks.  Fixed.